prime-angular-frontend/src/app/modules/adjudication/pages/organization-sites/organization-sites.component.ts

```diff
@@ -0,0 +1,192 @@
+import { AdjudicationRoutes } from '@adjudication/adjudication.routes';
+import { AdjudicationResource } from '@adjudication/shared/services/adjudication-resource.service';
+import { Component, OnInit } from '@angular/core';
+import { MatDialog } from '@angular/material/dialog';
+import { MatTableDataSource } from '@angular/material/table';
+import { ActivatedRoute, Router } from '@angular/router';
+import { PaginatedList } from '@core/models/paginated-list.model';
+import { SiteResource } from '@core/resources/site-resource.service';
+import { EmailUtils } from '@lib/utils/email-utils.class';
+import { RoutePath, RouteUtils } from '@lib/utils/route-utils.class';
+import { OrganizationAdminView } from '@registration/shared/models/organization.model';
+import { SiteRegistrationListViewModel } from '@registration/shared/models/site-registration.model';
+import { AssignAction, AssignActionEnum, ClaimNoteComponent, ClaimType } from '@shared/components/dialogs/content/claim-note/claim-note.component';
+import { ManualFlagNoteComponent } from '@shared/components/dialogs/content/manual-flag-note/manual-flag-note.component';
+import { SendEmailComponent } from '@shared/components/dialogs/content/send-email/send-email.component';
+import { DialogOptions } from '@shared/components/dialogs/dialog-options.model';
+import { CareSettingEnum } from '@shared/enums/care-setting.enum';
+import { SiteRegistrationNote } from '@shared/models/site-registration-note.model';
+import { concat, EMPTY, exhaustMap, forkJoin, map, noop, of, Subscription } from 'rxjs';
+
+@Component({
+  selector: 'app-organization-sites',
+  templateUrl: './organization-sites.component.html',
+  styleUrls: ['./organization-sites.component.scss']
+})
+export class OrganizationSitesComponent implements OnInit {
```

Comment:
Can `OrganizationSitesComponent` subclass `AbstractSiteAdminPage` so common code isn't duplicated?

Reply:
No, they serve slightly different function set, only one function in common and one similar function. Instead of making the logic more complicated to make them into one component, I think separating them into two will be easy to debug and easier to follow.

Reply:
OK, I saw `onAssign` and `onReassign` are virtually identical and mostly `onNotify` is the same but let's leave it for now